src/Compilers/CSharp/Portable/Symbols/PublicModel/PreprocessingSymbol.cs

```diff
@@ -103,6 +103,8 @@ ImmutableArray<SymbolDisplayPart> ISymbol.ToMinimalDisplayParts(SemanticModel se
 
         string ISymbol.MetadataName => _name;
 
+        int ISymbol.MetadataToken => 0;
```

Comment:
Are non-PE symbols covered in tests?

Reply:
There aren't explicit asserts that the value is 0, no. Will add some.

Reply:
Added a simple test. Not sure if there was anywhere else specifically these checks should be added?